### PR TITLE
python-utils-r1.eclass: Make epytest disable plugins we don't want + distutils-r1 installer bump

### PR DESCRIPTION
**Pull request by mgorny**
CC @gentoo/python

### CODE REVIEW

**Review by arthurzam — APPROVED** [review]

File: eclass/python-utils-r1.eclass [hunk 1]
```diff
@@ -1321,6 +1321,14 @@ epytest() {
 		-Wdefault
 		# override color output
 		"--color=${color}"
+		# disable the undesirable-dependency plugins by default to
+		# trigger missing argument strips.  strip options that require
```
Comment (sbraz) [inline eclass/python-utils-r1.eclass:1325]: Why 2 spaces here and on the next line?

File: eclass/python-utils-r1.eclass [hunk 2]
```diff
@@ -1321,6 +1321,14 @@ epytest() {
 		-Wdefault
 		# override color output
 		"--color=${color}"
+		# disable the undesirable-dependency plugins by default to
```
Comment (sbraz) [inline eclass/python-utils-r1.eclass:1324]: The dash in that word seems weird to me.

File: eclass/python-utils-r1.eclass (hunk 1 above)
Comment (thesamesam) [reply eclass/python-utils-r1.eclass:1325]: After the full stop? It's a an (older) style thing. I've never done it but some people learned it that way.

Comment (sbraz) [reply eclass/python-utils-r1.eclass:1325]: Ah OK, thanks, it seems weird to me (just like [this](https://www.python.org/dev/peps/pep-0008/#inline-comments)).

Comment (mgorny) [reply eclass/python-utils-r1.eclass:1325]: It makes spotting sentence boundaries easier.

Comment (SoapGentoo) [reply eclass/python-utils-r1.eclass:1325]: the yanks call is "french spacing" (eventhough IIRC France never did this). I also don't like it, but w/e

**Review by SoapGentoo — APPROVED** [review]

**Review by arthurzam — APPROVED** [review]

**Review by thesamesam — APPROVED** [review]

**Comment by thesamesam** [comment]
Merged.

**Pull request closed without merging**